whylog/tests/tests_constraints/test_user_constraint.py

```diff
@@ -34,8 +34,8 @@ def test_verify_fail(self):
 
 class TestTimeConstraint(TestCase):
     def setUp(self):
-        self.min_delta = timedelta(seconds=1)
-        self.max_delta = timedelta(seconds=10)
+        self.min_delta = 1
```

Comment:
Anyway, time unit should be parametrized with params_dict which should contain key unit with value like second, minute or hour.